src/psyclone/tests/test_files/gocean1p0/driver_test.f90

```diff
@@ -43,11 +43,11 @@ PROGRAM extract_example_with_various_variable_access_patterns
   use kind_params_mod
   use grid_mod
   use field_mod
-  use kernel_driver_test,  only: compute_kernel
+  use kernel_driver_test,  only: compute_kernel, compute_kernel_clash
```

Comment:
`compute_kernel_clash` is not used?

Reply:
Oops - I originally added a separate kernel to induce the name clash, before I realised that I can reuse the existing kernel. Time to get #706 (dummy library to allow compilation) done, that should have picked that up.